### PR TITLE
[DOCS-5987] SAML Add new way of configuring keystores

### PR DESCRIPTION
**Pull request by anxumalo**

### CODE REVIEW

File: saml-module/latest/install/index.md [hunk 1]
```diff
@@ -77,22 +103,37 @@ If you have not enforced SAML for a specific service provider, you can use the o
...
+       saml-keystore.password=change-me
+       saml-keystore.my-saml-key.password=change-me
+       saml-keystore.my-saml-key.algorithm=AES
        ```
 
-       Set the file permissions accordingly to limit who can read it.
+    4. (Deprecated) To continue using the old keystore configuration method:
 
-    4. Set the following values in the `alfresco-global.properties` file:
+        1. Generate a SAML keystore metadata file in the same location as the keystore and add the following content:
```
Comment (aforascu) [inline saml-module/latest/install/index.md:118]: this part should remain like it was before..I believe it was step 6.3:
aliases=my-saml-key
keystore.password=change-me
my-saml-key.password=change-me
These ones should be in the metadata file

**Review by aforascu — DISMISSED** [review]

File: saml-module/latest/install/index.md (hunk 1 above)
Comment (anxumalo) [reply saml-module/latest/install/index.md:114]: Reverted update in the next commit

**Review by gravitonian — APPROVED** [review]
Merge approve

**Review by oliveralfresco2018 — APPROVED** [review]
lgtm

**Pull request merged**